autoload/airline/extensions/coc.vim

```diff
@@ -38,7 +38,8 @@ function! airline#extensions#coc#get(type) abort
   if empty(cnt)
     return ''
   else
-    return (is_err ? s:error_symbol : s:warning_symbol).cnt
+    let lnum = printf('(L%d)', (info.lnums)[0])
```

Comment:
is guaranteed, that `info.lnums` always exists and it always contains at least one element?

Reply:
Yes, `info.lnums` will always exist. It defaults to a zero-filled array. Here's the reference commit: https://github.com/neoclide/coc.nvim/commit/e4cb34f7a17b42418341a440b52ec49e5ac304f6